modules/evaluation-server/src/Api/Program.cs

```diff
@@ -2,13 +2,33 @@
 using Serilog;
 using Serilog.Events;
 using Serilog.Formatting.Compact;
+using Serilog.Sinks.OpenTelemetry;
 
 try
 {
+    var otel_exporter = Environment.GetEnvironmentVariable("OTEL_EXPORTER_OTLP_ENDPOINT_GRPC") ?? "http://otel-collector:4317";
+
     Log.Logger = new LoggerConfiguration()
         .MinimumLevel.Override("Microsoft.AspNetCore", LogEventLevel.Warning)
         .Enrich.FromLogContext()
         .WriteTo.Console(new CompactJsonFormatter())
+        .WriteTo.OpenTelemetry(options =>
+        {
+            options.Endpoint = otel_exporter;
+
+            options.IncludedData = IncludedData.MessageTemplateTextAttribute
+                                | IncludedData.TraceIdField
+                                | IncludedData.SpanIdField
+                                | IncludedData.MessageTemplateMD5HashAttribute;
```

Comment:
I don't think it's necessary to include the template MD5 hash value in the logs, is there a specific purpose for this?

And it causes extra overhead:

https://github.com/serilog/serilog-sinks-opentelemetry/blob/main/src/Serilog.Sinks.OpenTelemetry/Sinks/OpenTelemetry/LogRecordBuilder.cs#L129

Reply:
Removed wtih commit # 7c02496